setup.py

```diff
@@ -69,6 +69,34 @@ def run(self):
 ssh = ["paramiko>=2.4.1"]
 all_remotes = gs + s3 + azure + ssh + oss
 
+# Extra dependecies to run tests
+tests_requirements = [
+    "PyInstaller==3.4",
+    "wheel>=0.31.1",
+    'futures>=3.2.0; python_version == "2.7"',
+    "pydot>=1.2.4",
+    'pathlib2==2.3.3; python_version == "2.7"',
```

Comment:
Already in https://github.com/iterative/dvc/pull/2017/files#diff-2eeaed663bd0d25b7e608891384b7298R118